storybook/cypress/e2e/date-picker-legacy.cy.ts

```diff
@@ -18,31 +18,46 @@ describe("Date Picker (Legacy)", () => {
   });
 
   it.skip("should have focus trap", () => {
-    cy.get("dso-date-picker-legacy.hydrated").as("date-picker-legacy").find("button.dso-date__toggle").click();
+    cy.get("dso-date-picker-legacy.hydrated").should("exist").find("button.dso-date__toggle").should("exist").click();
 
-    cy.get("@date-picker-legacy").find("select.dso-date__select--month").should("have.focus").realPress("Tab");
+    cy.get("dso-date-picker-legacy.hydrated").find(".dso-date__dialog :focus").should("have.focus");
 
-    cy.get("@date-picker-legacy").find("select.dso-date__select--year").should("have.focus").realPress("Tab");
+    cy.get("dso-date-picker-legacy.hydrated")
+      .find("select.dso-date__select--month")
+      .should("exist")
+      .and("have.focus")
+      .realPress("Tab");
+
+    cy.get("dso-date-picker-legacy.hydrated")
+      .find("select.dso-date__select--year")
+      .should("have.focus")
+      .realPress("Tab");
 
-    cy.get("@date-picker-legacy").find("button.dso-date__prev").should("have.focus").realPress("Tab");
+    cy.get("dso-date-picker-legacy.hydrated").find("button.dso-date__prev").should("have.focus").realPress("Tab");
 
-    cy.get("@date-picker-legacy").find("button.dso-date__next").should("have.focus").realPress("Tab");
+    cy.get("dso-date-picker-legacy.hydrated").find("button.dso-date__next").should("have.focus").realPress("Tab");
 
-    cy.get("@date-picker-legacy").find("button.dso-date__day.is-today").should("have.focus").realPress("Tab");
+    cy.get("dso-date-picker-legacy.hydrated")
+      .find("button.dso-date__day.is-today")
+      .should("have.focus")
+      .realPress("Tab");
 
-    cy.get("@date-picker-legacy").find("button.dso-date__close").should("have.focus").realPress("Tab");
+    cy.get("dso-date-picker-legacy.hydrated").find("button.dso-date__close").should("have.focus").realPress("Tab");
 
-    cy.get("@date-picker-legacy").find("select.dso-date__select--month").should("have.focus").realPress("Escape");
+    cy.get("dso-date-picker-legacy.hydrated")
+      .find("select.dso-date__select--month")
+      .should("have.focus")
+      .realPress("Escape");
 
-    cy.get("@date-picker-legacy").find("button.dso-date__toggle").should("have.focus");
+    cy.get("dso-date-picker-legacy.hydrated").find("button.dso-date__toggle").should("have.focus");
```

Comment:
Ik begrijp al deze changes niet zo. Dit doen we in een test die we skippen (`it.skip()`).

Volgens mij is het duidelijker als we deze changes ongedaan maken, dit zijn vermoed ik zo overblijfselen van een poging om deze test werkend te krijgen.

Reply:
Gedaan.

Er zijn wel nog 2 wijzigingen: deze heb ik in deze commit 31aeb024ea2acead15d96c45aaafe2f098c1f5a9 tijdens het visueel regressietesten opzetten, per abuis verwijderd.